spec/forms/serials_form_spec.rb

```diff
@@ -321,5 +321,39 @@
         expect(object_client).to have_received(:update).with(params: expected)
       end
     end
+
+    context 'when catalog links' do
+      let(:cocina_item) { build(:dro_with_metadata, id: druid, catkeys: ['6671606']) }
+
+      let(:expected) do
+        build(:dro_with_metadata, id: druid).new(
+          description: {
+            title: [
+              {
+                structuredValue: [
+                  { value: 'factory DRO title', type: 'main title' }
+                ]
+              }
+            ],
+            purl: 'https://purl.stanford.edu/bc123df4567'
+          },
+          identification: {
+            catalogLinks: [
+              { catalog: 'symphony', refresh: false, catalogRecordId: '6671606' }
```

Comment:
It's kind of too bad all the boilerplate is needed to just test `refresh: false`?